packages/ai/core/tool/mcp/types.ts

```diff
@@ -139,13 +140,40 @@ export const JSONRPCMessageSchema = z.union([
 ]);
 export type JSONRPCMessage = z.infer<typeof JSONRPCMessageSchema>;
 
+/**
+ * Transport interface for MCP (Model Context Protocol) communication.
+ */
```

Comment:
Is this similar to what the original MCP library has? if so might be worth leaving a // note for developers